i6_models/parts/conformer/mhsa.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Optional, Callable
+import torch
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class ConformerMHSAV1Config(ModelConfiguration):
```

Comment:
Please add documentation as stated in https://github.com/rwth-i6/i6_models/pull/8#discussion_r1192185282 so it shows up in the IDE.